src/SourceNodes/sourcenode.js

```diff
@@ -385,7 +385,6 @@ class SourceNode extends GraphNode {
         }
 
         if (currentTime >= this._stopTime) {
-            clearTexture(this._gl, this._texture);
```

Comment:
what's the idea behind removing this line?

Reply:
Right now we have a black frame between each videos (aka sections with a video node). The cause of this, is this line clearing the texture.

So we have two choices from my understanding:
- First option, we have a flag in the sourcenode to know if its the last video node of our layer, and if it is we clear the texture.
From a code pov inside studio it means we need to loop on every section chains and if it the last video node we add this flag.

- Second option, we always remove this black frame (this is what I have done here)

Let me know if it makes sense.